packages/api-bindings/src/graphql/common.graphql

```diff
@@ -172,26 +172,11 @@ fragment PublicationStats on PublicationStats {
 fragment MirrorBase on Mirror {
   __typename
   id
-  stats {
-    ...PublicationStats
-  }
-  metadata {
-    ...Metadata
-  }
+  createdAt
   profile {
     ...Profile
   }
-  __collectModule: collectModule {
-    ...CollectModule
-  }
-  createdAt
-  hidden
-  isGated
-  reaction(request: { profileId: $observerId })
-  hasCollectedByMe(isFinalisedOnChain: true)
-
-  hasOptimisticCollectedByMe @client
-  collectPolicy @client
```

Comment:
for consistency, could you also apply the same changes on `MirrorBase` fragment in `packages/client/src/graphql/fragments.graphql`?

Reply:
For posterity: agreed to look at this holistically as part of the low-level client effort.